pages/react-canvas-components.mdx

```diff
@@ -0,0 +1,256 @@
+import { RandomHexagons } from '../components/hexagons/RandomHexagons';
+import { EditOnCodeSandbox } from '../components/EditOnCodeSandbox';
+import { css } from '@emotion/core';
+
+export const meta = {
+  title: 'React + Canvas = 💜',
+  description: 'an untold love story',
+  date: 'January 12, 2019',
+};
+
+The `<canvas/>` HTML element can be used to draw graphics with a finer control than the usual DOM or SVG. But with React, trying to draw on a canvas is not intuitive as their interfaces are quite different. With React, each component own their node contrary to canvas where there is only one shared node, and we draw in it. Let's see how we can make a canvas visualization with React components !
+
+## 🧑‍🏫 Canvas 101
+
+The `canvas` element is like a sheet of paper. To draw in real life, you would take a pen, move your hand to a first position, and draw a line by moving your hand to another position. The browser API to draw on a canvas is actually very similar. We first need to make a blueprint of the shape we want to draw (like using a wood pencil), and then we can color it.
```

Comment:
- ".. in real life you would" (i.e. remove the comma)
- "wood pencil" - "real pencil" is probably better
- "and then we can color it" - "that can later be colored in" 
-

Reply:
re: "wood pencil" I meant something that does not really leaves a trace, that you can erase if you want. What would be a good way of describing that ?

Reply:
You can always do that with a pencil, whether it's wood or not :)